<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Auteurs</class>
 <widget class="QDialog" name="Auteurs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auteurs</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>781</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox {
border: 2px solid black;
border-radius: 4px;
}</string>
   </property>
   <property name="title">
    <string>Ajouter Nouvelle Auteur :</string>
   </property>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nom Auteur :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_auteur">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_ajouter">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#bt_ajouter{
color:black;
background-color:#a0a0a0;
border-width:0px;
border-radius:4px;
}
#bt_ajouter:hover{

background-color:#27a9e3;

}</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_auteur">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>681</width>
      <height>271</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>135</number>
    </attribute>
    <column>
     <property name="text">
      <string>ID AUTEUR</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>LIBELLE AUTEUR</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DATE NAISSANCE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Modifier</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Date de Naissance :</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dt_naissance">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="ck_modifer">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>190</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_modifier">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#bt_modifier{
color:black;
background-color:#a0a0a0;
border-width:0px;
border-radius:4px;
}
#bt_modifier:hover{

background-color:#27a9e3;

}</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Consulter Auteur</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/logo.png);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
